src/summary/SummaryMetadata.jsx

```diff
@@ -1000,7 +1002,11 @@ export default class SummaryMetadata {
     }
 
     getMissingCriteriaSubsectionName = () => {
-        return `Missing ${this.trialDisplayMissingCriteria} Criteria`;
+        return (this.trialDisplayMissingCriteria !== "") ? (`Missing ${this.trialDisplayMissingCriteria} Criteria`) : "Missing Criteria";
+    }
+
+    getMissingCriteriaDisplay = () => {
+        return (this.trialDisplayMissingCriteria !== "") ? true : false;

```

Comment:
I think this line can be changed to ```return this.trialDisplayMissingCriteria !== "";```

Seems kind of redundant to use a ternary operator here.